examples/dev-values-keycloak-auth.yaml

```diff
@@ -64,6 +64,8 @@ keycloak:
   auth:
     adminUser: "admin"
     adminPassword: "F3LVqnxqMmkCQkvyPdJiwXodqQncK@"
+  postgresql:
+    postgresqlPassword: "xinf-2-doHTG._sMJi3*kZgQAabuet"
```

Comment:
In a recent test, this did not work as expected. Be sure to deploy this branch with this example before merging this PR.

Reply:
Removed this commit for now. Will revisit later.